build.gradle

```diff
@@ -62,6 +63,9 @@ dependencies {
   implementation "org.scala-lang:scala-library:${scalaBinaryVersion}"
   implementation 'org.scala-lang.modules:scala-parallel-collections_2.13:1.0.4'
 
+  /* CORE Akka */
+  implementation "com.typesafe.akka:akka-actor_${scalaVersion}:${akkaVersion}"
```

Comment:
You can make this a `testImplementation` because it's not used with the main code. This way, we can keep the size of shadow jars down